templates/realmcreator.yaml

```diff
@@ -0,0 +1,55 @@
+{{ if .Values.realmCreator.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "keycloak.fullname" . }}-realm-creator-configmap
+data:
+  realm.json: |-
+    {{- .Values.realmCreator.options | nindent 4 }}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "keycloak.fullname" . }}-realm-creator
+  annotations:
+    "helm.sh/hook": post-install, post-rollback, post-upgrade
+    "helm.sh/hook-weight": {{ .hookWeight | default 5 | quote }}
+spec:
+  template:
+    metadata:
+      name: {{ include "keycloak.fullname" . }}-realm-creator
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: realm-creator
+        image: adorsys/keycloak-config-cli
```

Comment:
where is the tag?
and does it make sense to be configurable?

Reply:
It is now configurable with some defaults set